plugins/catalog-backend-module-github/src/providers/GithubMultiOrgEntityProvider.ts

```diff
@@ -301,9 +815,11 @@ export class GithubMultiOrgEntityProvider implements EntityProvider {
     team: GithubTeam,
     ctx: TransformerContext,
   ): Promise<GroupEntity | undefined> {
-    const result = this.options.teamTransformer
-      ? await this.options.teamTransformer(team, ctx)
-      : await defaultOrganizationTeamTransformer(team, ctx);
+    if (this.options.teamTransformer) {
+      return await this.options.teamTransformer(team, ctx);
```

Comment:
ah this is aligning with actual expectations since the custom transformer should override the default one - since this is technically breaking, I'll make this as a minor release